### DISCUSSION TITLE
setting up first noob questions

### DISCUSSION THREAD

**Discussion by Wichipingui** [5andr0/StockChecker#2] [Q&A]
Hi, Im also on the boat of getting one GTX 3060Ti for under 450 EUR, and I have some questions because im new to this stuff (Git, etc).
1. Can I set in config.ini the frequency as a decimal number? for example 0.1 ? so it would check every 6 seconds. Whats the minimum I can get here? I read that scrapers tools check every second or even faster.
2. I only downloaded this from git: StockChecker-v0.9.1-win-x64 is this correct or should I also download "StockChecker-main"? If so, where I put those files?
3. For checking only 3060Ti under 450 EUR and assuming "2704136, 2704490, 2701220, 2702392, 2703630, 2704889, 2701437" are the IDs for the 3060Ti under 450 EUR. should I edit this:

[collections]
# set to 1 to fetch all key-matching RTX cards from www.mediamarkt.de/de/campaign/grafikkarten-nvidia-geforce-rtx-30.html
NVIDIA_RTX_3060_Ti = 1

to look like this?

[collections]
# set to 1 to fetch all key-matching RTX cards from www.mediamarkt.de/de/campaign/grafikkarten-nvidia-geforce-rtx-30.html
NVIDIA_RTX_3060_Ti = [2704136, 2704490, 2701220, 2702392, 2703630, 2704889, 2701437]

Thank you!!

**Comment by 5andr0** [comment]
Hi, thanks for your interest!
Frequency in full minutes was intended to avoid IP bans.
I added a _max_price_ option for you. Just add
```
[general]
max_price=450.00
[collection]
NVIDIA_RTX_3060_Ti = 1
```
Download the latest binaries [here](https://github.com/5andr0/StockChecker/releases/tag/v0.9.2)

This is how it could've been done:
Use a custom collection:
```
[collection]
# don't use all cards from the 3060 Ti collection, only check the budget_cards collection
NVIDIA_RTX_3060_Ti = 0
budget_cards = [2704136, 2704490, 2701220, 2702392, 2703630, 2704889, 2701437]
```

or use all _NVIDIA_RTX_3060_Ti_ ids but ignore those with a higher price:
```
[collection]
# use all cards from the 3060 Ti collection but ignore all cards with a high price from the ignore collection
NVIDIA_RTX_3060_Ti = 1
ignore = [list of 3060 Ti ids with a price tag > 450 €]
```

**Reply by Wichipingui** [reply]
Hi, thank you very much for your quick response!
The problem is that with a frequency in minutes I have no chance against scalpers. they are way too fast and I need something that at least every 5 seconds check. Would that be possible?
PS: Happy new year!
